### PR TITLE
Link to skimage lesson by Data Carpentry.

### PR DESCRIPTION
**Pull request by mkcor**
@stefanv what do you think about mentioning it on our homepage?

References:

* https://carpentries.org/blog/2023/01/dc-image-processing-stable-release/
* https://forum.image.sc/t/data-carpentry-image-processing-with-python-has-been-released/76597

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -85,5 +85,7 @@ visit our `gallery </docs/dev/auto_examples>`__.
          :class: coins-sample span6
 
 You can read more in our `user guide </docs/stable/user_guide.html>`__.
+For a proper introduction to image processing, you may want to check out this
```
Comment (stefanv) [inline index.rst:88]: Perhaps:

For an introduction to image processing using scikit-image, see
`this lesson <...>` by Data Carpentry.

**Comment by stefanv** [comment]
+1, I just suggested a slightly different wording, but not super important.

File: index.rst (hunk 1 above)
Comment (mkcor) [reply index.rst]: Yes, your wording is more neutral, thanks! Applied: f57948eb8eb62

**Comment by stefanv** [comment]
Thanks, also for your work on that lecture!

**Pull request merged**